src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -531,7 +542,7 @@ export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
         >
           <WithLazyMount shouldUse={Boolean(receiveAddress)} component={RecipientAddress} />
           <WithLazyMount
-            shouldUse={hasUserEnteredAmount && !walletSupportsBuyAssetChain}
```

Comment:
This removal isn't a mistake:

1. feels a bit weird for this not to be shown initially from a UX perspective
2. this is wrongly reactive when there are no buy accounts and the trade buy asset doesn't display, and will cause issues